bin/extract_galaxy_tools.py

```diff
@@ -28,15 +28,16 @@
 # BIOTOOLS_API_URL = "https://130.226.25.21"
 
 USEGALAXY_SERVER_URLS = {
-    "UseGalaxy.org": "https://usegalaxy.org",
+    "UseGalaxy.org (Main)": "https://usegalaxy.org",
```

Comment:
I would remove `Main` here

Reply:
it is the name you get from https://galaxyproject.org/use/feed.json , so I use this one to avoid dublication